File: doc/src/codes/Users_Manual/gain/group_0.cs
using AUTD3Sharp.Utils;
using AUTD3Sharp.Gain;

var g = new Group((dev, tr) => tr.Idx <= 100 ? "null" : "focus")
          .Set("null", new Null())
          .Set("focus", new Focus(new Vector3d(x, y, z)));

File: doc/src/codes/Users_Manual/gain/group_0.cpp
const auto g =
    autd3::gain::Group(
        [](const autd3::Device& dev,
           const autd3::Transducer& tr) -> std::optional<const char*> {
          if (tr.idx() <= 100) return "null";
          return "focus";
        })
        .set("null", autd3::gain::Null())
        .set("focus", autd3::gain::Focus(autd3::Vector3(x, y, z)));
